### ISSUE TITLE
Include key when overriding default style?

### ISSUE THREAD

**Issue by TysonMN** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2143]
The [Overriding Default Styles wiki page](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Overriding-Default-Styles) gives an example like the following for how to override a default style.
```xaml
<Style x:Key="MaterialDesignFloatingHintTextBox" BasedOn="{StaticResource MaterialDesignFloatingHintTextBox}" TargetType="{x:Type TextBox}">
    <Setter Property="FontSize" Value="24" />
</Style>
```

However, I would not describe this as overriding a default style.  I would describe this as overriding a non-default style because a key is given.

Instead, I expected an example of how to override a default style to be more like this.
```xaml
<Style BasedOn="{StaticResource MaterialDesignTextBox}" TargetType="{x:Type TextBox}">
    <Setter Property="FontSize" Value="24" />
</Style>
```

MDinXAML assigns a default style for `TextBox` [here](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/d75bac11b6d25ad34c78e416b26727344da8c487/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Defaults.xaml#L80).  The main thing to observe in both cases is that neither style is identified with a key.  This means that the style is the default style for the target type.  Of course the actual default style ends up being whichever style is processed last.

Can this wiki page be extended to include two examples: one that shows how to override a non-default style defined by MDinXAML (which could be the one already on that wiki page) and another one that shows how to override a default style?

If so, I am willing to make this change.

I encountered this confusion in the process of writing https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/issues/922#issuecomment-712927361.

**Comment by Keboo** [comment]
Hi @TysonMN sorry for the long delay on this. Yes I agree the wording is misleading. Feel free to update the wiki as you suggested!

**Comment by TysonMN** [comment]
I updated the wiki.  In addition to adding the example, I changed the title as well.  Then I updated the reference to that page in the sidebar.

Here is that content on its newly renamed page:
https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Overriding-Material-Design-Styles

Here are diffs of my changes:
- [sidebar revision](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/_Sidebar/_compare/dbd1e71e3e946220dc7b6e105344dcbc6f93791d...14c191c5da46147baf9a267345a3be5e3533a583)
- [revision of page excluding change of title](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/wiki/Overriding-Material-Design-Styles/_compare/6a16dd66f282240ed021345338c923372458f490...a28de1209f0bd74b558c4c7ac165fd5c8acd4a30)

When I first made the change, I changed the text and changed the title in the same edit.  That was not ideal since there isn't a diff of the text in that case.  So I undid and then redid my changes to the text.  That is how I arrived at the above diff.
